src/releng_notification_identity/requirements.txt

```diff
@@ -0,0 +1,2 @@
+-e ./../../lib/cli_common #egg=mozilla-cli-common
```

Comment:
can you change this line to ``-e ./../../lib/cli_common[sentry] #egg=mozilla-cli-common`` and then run ``nix-shell nix/update.nix --argstr pkg releng-notification-identity`` otherwise it will fail in staging since there is no raven package and sentry is already enabled in staging.